lib/epics/xe3.rb

```diff
@@ -0,0 +1,5 @@
+class Epics::XE3 < Epics::CDD
```

Comment:
You might still be working on it, as no review has been requested so far.

One thing I noticed is that compared to the other order types, no unit test was added for this one

See https://github.com/railslove/epics/blob/211677a859c5fe6c54952fddff8ddfc3d29f1b45/spec/orders/cdd_spec.rb for reference